test/util/solidHelper.ts

```diff
@@ -1,17 +1,52 @@
 import Path from "path";
 import {AppRunner} from "@solid/community-server";
+import {Session} from "@rubensworks/solid-client-authn-isomorphic";
 import {readFileSync} from "fs";
 import {Store} from "n3";
 import {stringToStore} from "../../src/util/Conversion";
+import {login, isLoggedin, getSession, RegistrationType} from "../../src/util/Login";
 
 /***************************************
  * Title: solidHelper.ts
  * Description: Helper functions for setting up the test environment
  * Author: Wout Slabbinck (wout.slabbinck@ugent.be)
  * Created on 28/03/2022
  *****************************************/
-const {port} = require('./testconfig.json')
+const {port, authport} = require('./testconfig.json')
 export const baseUrl = `http://localhost:${port}/`
+export const authBaseUrl = `http://localhost:${authport}/`
+
+/**
+ * Login to the auth solid-server
+ */
+export async function initAuth() {
+    const validatedOptions = {
+        applicationName: "LDES-orchestrator",
+        registrationType: RegistrationType.Dynamic,
+        solidIdentityProvider: authBaseUrl
+    };
+    console.log("Login with email: test@mail.com password: test");
+    await login(validatedOptions);
+    await isLoggedin(); // code that checks whether you are already logged in
+}
+
+export async function createTestAcc() {
```

Comment:
Isn't this done by the test script now?

Reply:
This `initAuth` function is still needed, but the createTestAcc is replaced by the script jes,

Reply:
Oh, I did not know that. Then only one of them can be remove.